src/pages/extending-chat-widget/chat-widget-adapters/adapters-core/index.mdx

```diff
@@ -0,0 +1,83 @@
+---
+weight: 10
+category: "extending-chat-widget"
+subcategory: "chat-widget-adapters"
+title: "Adapters core"
```

Comment:
I'm not really sure about the title. How about?

The core of Adapters
The Adapter core
The core implementation

Reply:
"Adapters core" means here that we describe the core implementation of each adapter. Additionally it is much shorter for title and navigation item

Reply:
It would need an apostrophe, Adapters' core, which we cannot put in the headline. 
`Adapters core` is incorrect, it should not be a plural form. Just like we say Developer Program, not Developers Program, boat race not boats race, etc. The noun works as an adjective.

`The Adapter core` would be correct and would translate into "core adapterowy" implying that all adapters share the same core. We could skip "the" for the sake of brevity. So, my suggestion is "Adapter core". Another option would be `Adapter cores` suggesting each adapter has a different core.